api/WorkItemTrackingApi.ts

```diff
@@ -1146,7 +1149,7 @@ export class WorkItemTrackingApi extends basem.ClientApiBase implements IWorkIte
         customHeaders = customHeaders || {};
         customHeaders["Content-Type"] = "application/json-patch+json";
 
-        this.vsoClient.getVersioningData("2.2-preview.2", "wit", "72c7ddf8-2cdc-4f60-90cd-ab71c14a399b", routeValues, queryValues)
+        this.vsoClient.getVersioningData("2.2-preview.2", "wit", "62d3d110-0047-428c-ad3c-4fe872c91c74", routeValues, queryValues)
```

Comment:
Not sure about the deprecation but I have taken the route from the C# controller.